cmd/admiralctl/admiralctl.go

```diff
@@ -0,0 +1,31 @@
+/*
+Copyright 2023 The KubeAdmiral Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"k8s.io/component-base/cli"
+	"k8s.io/kubectl/pkg/cmd/util"
+
+	"github.com/kubewharf/kubeadmiral/pkg/admiralctl"
+)
+
+func main() {
+	cmd := admiralctl.NewAdmiralCtlCommand("admiralctl", "admiralctl")
```

Comment:
Why we need to paas `admiralctl` parameters? 
```suggestion
	cmd := admiralctl.NewDefaultAdmiralctlCommand()
```